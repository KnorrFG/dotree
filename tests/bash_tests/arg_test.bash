$DT -c arg_test.dt f alpha beta gamma

printf "lala\n" | $DT -c arg_test.dt f alpha beta

printf "\n" | $DT -c arg_test.dt f alpha beta

printf "\n\n\n" | $DT -c arg_test.dt f

