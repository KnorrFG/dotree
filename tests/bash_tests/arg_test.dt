menu root {
	f: cmd {
		vars a, b, c="foo"
		"echo $a $b $c"
	}
}
