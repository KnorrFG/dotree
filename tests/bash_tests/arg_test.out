[?25l[?25hecho $a $b $c
alpha beta gamma
[?25l[?25h[1A[2K[1B[1Aecho $a $b $c
alpha beta lala
[?25l[?25h[1A[2K[1B[1Aecho $a $b $c
alpha beta foo
[?25l[?25h[3A[2K[1B[2K[1B[2K[1B[3Aecho $a $b $c
foo
